pkg/util/util.go

```diff
@@ -1091,7 +1091,25 @@ func IsValidS3BucketName(name string) bool {
 	return validBucketNameRegex.MatchString(name)
 }
 
-// GetFlagStringOrPrompt returns flag value but if empty will promtp to read from stdin
+// GetFlaIntOrPrompt returns flag value but if empty will prompt to read from stdin
+func GetFlaIntOrPrompt(cmd *cobra.Command, flag string) int {
+	val, _ := cmd.Flags().GetInt(flag)
+	if val != -1 {
+		return val
+	}
+	fmt.Printf("Enter %s: ", flag)
+	_, err := fmt.Scan(&val)
+	if strings.Contains(err.Error(), "expected integer") {
```

Comment:
do we want to test here for negative numbers?

Reply:
It is general util, We cannot assume that a negative is forbidden.
The main reason for this error handling is to avoid unclean exit with panic when we expect an integer and we get a string.

Reply:
so you can test it in the cli?